src/core/backend_context.cc

```diff
@@ -946,26 +1015,55 @@ BackendInputCollector::SetFixedSizeInputTensor(
     }
 
     // Direct copy without intermediate pinned memory.
```

Comment:
Isn't this case the same discussion we already had where we don't want to async (for now) when it is direct GPU<->CPU copy that is not using a pinned buffer?

Reply:
Fixed